docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
+be created using the :class:`~gcloud.monitoring.metric.MetricDescriptor` class or be
+auto-created.  For more information on creating metric descriptors, see the `Metric Descriptors`_
+section. For information on metric type auto-creation, see `Auto-creation of custom metrics`_.
+
+You must also create a fully parameterized :class:`~gcloud.monitoring.resource.Resource` from one
+of the available resource descriptors which can be listed using the
+:class:`~gcloud.monitoring.resource.ResourceDescriptor` class, which similarly requires that its
+available labels have values specified. The ``project_id`` label is a special case that should
+not be specified.
+
+>>> from gcloud import monitoring
+>>> # Create a Resource object for the desired monitored resource type.
+>>> resource = client.resource('gce_instance', labels={
+...     'instance_id': '1234567890123456789',
+...     'zone': 'us-central1-f'
+... })
+>>> # Create a Metric object, specifying the metric type as well as values for any metric labels.
+>>> metric = client.metric(type='custom.googleapis.com/my_metric', labels={
+...      'status': 'successful'
+... })
+
+Please refer to the `Metrics`_ documentation for more information.
+
+With a ``Metric`` and ``Resource`` specified, the :class:`~gcloud.monitoring.client.Client`
+can be used to write :class:`~gcloud.monitoring.timeseries.Point` values.
+
+When writing points, the Python type of the value must match the *value_type* specified
+in the type of the associated `Metric Descriptors`_. For example, a Python float will map to
+``ValueType.DOUBLE``.
+
+Stackdriver Monitoring supports several *metric kinds*: `GAUGE`, `CUMULATIVE`, and `DELTA`.
+However, `DELTA` custom metrics are not supported.
+
+*GAUGE* metrics represent only a single point in time, so only the ``end_time`` should be
+specified::
+
+    >>> client.write_point(metric=metric, resource=resource, 3.14, end_time=end) # API call
+
+By default, ``end_time`` defaults to :meth:`~datetime.datetime.utcnow()`, so metrics can be written
+to the current time as follows::
+
+   >>> client.write_point(metric, resource, 3.14) # API call
+
+*CUMULATIVE* metrics enable the monitoring system to compute rates of increase on metrics that
+sometimes reset, such as after a process restart. Without cumulative metrics, this
+reset would otherwise show up as a huge negative spike.
+
+For *CUMULATIVE* metrics, the same start time should be re-used repeatedly as more points are
+written to the time series. In this context, the start time is also called the *reset*
+time. Once the cumulative metric is ready to be reset, the *reset* time should be reset by
```

Comment:
I don't understand what it would mean for a cumulative metric to be "ready to be reset".

This paragraph and the previous one need to be combined and tightened up.